source/about/self-hosted-subscriptions.rst

```diff
@@ -0,0 +1,170 @@
+
+You can select **Renew license now** to begin the renewal process. You can also select the **x** to dismiss the notification. The notification is reactivated when your browser is refreshed or you reload the Mattermost Desktop App.
+
+How long does it take to renew a license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you’ve started the renewal process, we'll be in contact with you to confirm your order and send you the order form. There may be additional paperwork required. Once we have the signed order form and (if applicable) the necessary paperwork from you, we're able to process the renewal and issue your license key within 24 hours.
+
+What happens to my license if I don't renew in time?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't renew within the 60-day renewal period, a 10-day grace period is provided for you to upload a new license key. During this period your Mattermost installation runs as normal, with full access to commercial features. During the grace period, the notification banner is not dismissable.
+
+When the grace period expires, your Professional or Enterprise plan is downgraded to the Starter plan and other plan features are disabled.
+ 
+What happens when my license expires?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't renew your license within the 10-day grace period, your Mattermost version is automatically downgraded to Starter plan so you can still access and use Mattermost. However, subscription features will no longer be available, and if you are currently using them, the functionality will no longer be accessible.
+
+When you renew, the subscription features will become available with the previous configuration (provided no action such as user migration has been taken).
+
+Why can't I dismiss the expiry notification banner?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If there's a red expiry announcement banner stating: "Enterprise license is expired and some features may be disabled. Please contact your System Administrator for details." it means your grace period has expired. This announcement banner persists until the license is renewed, and is visible to all users.
+
+Once a new license is applied, the banner will no longer be visible.
+
+If you don't plan to renew your subscription, revoke the expired license in **System Console > Edition and License**.
+
+Can other members of my organization use the Customer Portal account to manage our subscription?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We currently support a single account/user per organization. The ability to add multiple users per organization will be available in a future release.
+
+Where can I find the license agreement for Mattermost Enterprise Edition?
```

Comment:
Are we leaving "Edition" in here?

Reply:
Yes, because this is the license for the binary "Mattermost Enterprise Edition".